app/lua/session.go

```diff
@@ -316,10 +314,34 @@ func GetFlash(L *lua.LState) int {
 		return 0
 	}
 
-	// Get value from the flash map
-	v, ok := session[key.String()].(string)
+	// Get session data from the user data field
+	session := getSessionData(L)
 
-	if !ok {
+	// Get element from session
+	val := session[key.String()]
```

Comment:
What if the key does not exists? will it panic? maybe we should add `val, ok := ...`